src/index.ts

```diff
@@ -0,0 +1,99 @@
+import { getDependencies } from './getdependencies';
+import { getRepoIds } from './getrepos';
+const JupiterOneClient = require('@jupiterone/jupiterone-client-nodejs');
+const gql = require("graphql-tag");
+
+const J1_USER_POOL_ID_PROD = "us-east-2_9fnMVHuxD";
+const J1_CLIENT_ID_PROD = "1hcv141pqth5f49df7o28ngq1u";
+const accessToken = '';
+
+let success = 0, failure = 0;
+const missingDeps = [];
+
+;(async () => {
+  const j1Client = await new JupiterOneClient({
+    account: 'j1dev',
+    username: 'j1dev',
+    password: 'dev',
+    poolId: J1_USER_POOL_ID_PROD,
+    clientId: J1_CLIENT_ID_PROD,
+    accessToken: accessToken,
+    dev: true
+  }).init();
+  const repoMap = await getRepoIds();
+  
+  //value = Repo Name, key = Repo Entity ID
```

Comment:
If you have a comment mapping a variable name to a more descriptive name, you should probably just use a better variable name.

Reply:
Sorry about that was just to help me keep things straight. I meant to delete that.

Reply:
Well, I don't only mean that I want the comment deleted. I mean that you should probably rename the variables. :grinning: